dpctl/_sycl_queue.pyx

```diff
@@ -239,16 +319,14 @@ cdef class SyclQueue(_SyclQueue):
                 raise SyclQueueCreationError(
                     "SYCL Context could not be created from '{}'.".format(arg)
                 )
-            elif status == -4:
+            elif status == -4 or status == -6:
```

Comment:
This is a non-documentation related change. Better to move to a separate commit.

Reply:
Done, moved to #389